### ISSUE TITLE
Determine if branch has been merged

### ISSUE THREAD

**Issue by salcode** [ironcodestudio/ironcode-git-enhancements#110]
I'd like a way to determine if a branch has been merged into another branch. I'm not sure that this has a lot of stand-alone use however I think it would be helpful in the context of more complex scripts.

**Comment by salcode** [comment]
## Manually

If you are doing this manually, using 

```
git branch --merged
```

will give you a list of all branches that have been merged into your current branch.

## Programmatically

While it is nice to be able to do this manually (and we could probably do some fancy grepping to get the answer), I think there is another approach.

```
git merge-base mybranch develop
```

Will give me the most recent common commit between `mybranch` and `develop`. If we compare this most recent commit on `mybranch`, if they are the same we'll know that `develop` is a superset of `mybranch` and thus `mybranch` has been merged into `develop`.

In other words, if 

```
git merge-base mybranch develop
```

and

```
git rev-parse mybranch
```

return the same hash, then `mybranch` has been merged into `develop`

**Comment by salcode** [comment]
Alternatively, we could do

```
git log develop..mybranch
```

If there is **no output** then `mybranch` has no commits that do not appear in `develop` (i.e. it `mybranch` has been merged into `develop`), if there is **any output** then `mybranch` has commits NOT in `develop` (`mybranch` has no been merged into `develop`)
